library/globals.inc.php

```diff
@@ -1372,6 +1372,12 @@
       '',                               // default
       xl('Email address, if any, to receive emergency login user activation messages.')
     ),
+     'new_validate' => array(
```

Comment:
@sunsetsystems is this really necessary? We have added several global configuration parameters into codebase , and there was no such requirements. It's a bit strange when two code reviewers have different naming rules for parameters.